packages/hw-app-xrp/src/Xrp.js

```diff
@@ -0,0 +1,180 @@
+//@flow
+
+import type Transport from "@ledgerhq/hw-transport";
+import BIPPath from "bip32-path";
+
+/**
+ * Ripple API
+ *
+ * @example
+ * import Xrp from "@ledgerhq/hw-app-xrp";
+ * const xrp = new Xrp(transport);
+ */
+export default class Xrp {
+  transport: Transport<*>;
+
+  constructor(transport: Transport<*>) {
+    this.transport = transport;
+    transport.setScrambleKey("w0w");
+  }
+
+  /**
+   * get Ripple address for a given BIP 32 path.
+   *
+   * @param path a path in BIP 32 format
+   * @param display optionally enable or not the display
+   * @param chainCode optionally enable or not the chainCode request
+   * @param ed25519 optionally enable or not the ed25519 curve (secp256k1 is default)
+   * @return an object with a publicKey, address and (optionally) chainCode
+   * @example
+   * const result = await xrp.getAddress("44'/144'/0'/0/0");
+   *
+   * {
+   *   "publicKey": "0324E5F600B52BB3D9246D49C4AB1722BA7F32B7A3E4F9F2B8A1A28B9118CC36C4",
+   *   "address": "rPwReL1dVYnVzFKShJhBxNPSbfX7rjJQEd"
```

Comment:
we might want to remove any "real" address from the code sample, just so people don't accidentally use them like it happened in the past and then contact us to reclaim them

probably you can just show a `const { publicKey, address } = result;`